projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -143,7 +122,7 @@ const Inbox = () => {
 			>
 				{ () => <InboxView /> }
 			</TabPanel>
-		</Layout>
```

Comment:
In this file: 
* We can remove Layout because I'm using react routers Outlet instead. 
* We no longer need a lot of the header content.